template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-FSharp/.template.config/template.json

```diff
@@ -116,7 +116,8 @@
         }
       ],
       "defaultValue": "Default",
-      "description": "Select the SDK extensions profile. More information at https://aka.ms/mstest/sdk/extensions-profile"
+      "description": "Select the SDK extensions profile. More information at https://aka.ms/mstest/sdk/extensions-profile",
+      "replaces": "$(ExtensionsProfile)"
```

Comment:
I see that not all properties of `ExtensionsProfile` for F# are aligned with C#/VB. Could you fix that too please?

Reply:
Fixed it.